scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java

```diff
@@ -2099,7 +2099,7 @@ List<JobLabelInfo> createLabels(List<String> labels)
      */
     @RoleAdmin
     List<JobLabelInfo> setLabels(List<String> labels)
-            throws NotConnectedException, PermissionException, LabelValidationException;
+            throws NotConnectedException, PermissionException, LabelValidationException, LabelConflictException;
```

Comment:
why not reuse LabelValidationException ?

Reply:
It seemed to me that it's not really a label validation exception, which is thrown when there is a formatting issue on a label (special characters).
Should I use the LabelValidationException ?